coursedata/achievements/nb_NO.yaml

```diff
@@ -9,7 +9,7 @@ adventure_is_worthwhile:
     image: prestasjoner.png
     text: "Lagre et program fra et eventyr\n"
 programming_protagonist:
-    title: Programmerings Protagonist
+    title: Programmeringshelten
     image: prestasjon.png
```

Comment:
This image string should not be translated! It is the name of the image file and it won't work when being translated: should be reversed too `achievement.png`.

Reply:
oufh, maybe not the best idea to have it as a translatable string then?

Reply:
I will fix this in weblate 😁

Reply:
Agree, we should look into the current YAML structure and maybe shift the image route to another location. Will make an issue for this to figure it out.

Reply:
This is fixed! Thanks @Kegulf !